widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/IsMessageCenterAvailableUseCase.kt

```diff
@@ -34,7 +34,8 @@ class IsMessageCenterAvailableUseCase(
     fun containsMessagingQueue(queues: Array<Queue>): Boolean {
         val messagingQueues = queues
             .filter { queue -> queue.id == queueId }
-            .filter { queue -> queue.state.status == QueueState.Status.OPEN }
+            .filterNot { queue -> queue.state.status == QueueState.Status.CLOSED }
+            .filterNot { queue -> queue.state.status == QueueState.Status.UNKNOWN }
```

Comment:
You just blew my mind! I have never known that there is `filterNot` method. And the possibility never even crossed my mind!!!